src/components/CompanySelect/index.tsx

```diff
@@ -56,7 +56,7 @@ export function CompanySelectUI(props: CompanySelectUIProps): ReactElement {
 
     return (
         <div
-            className={`shadow-md bg-white rounded-lg w-72 overflow-hidden dark:bg-bluegray-6 company-select ${className}`}
+            className={`shadow-md bg-white rounded-lg w-[271px] overflow-hidden dark:bg-bluegray-6 company-select ${className}`}
```

Comment:
The addition of the recording form [blue] button has made this open too far to the edge of the component, so `271` vs `288` (`w-72`)